eventActions/tosReminderAction.js

```diff
@@ -0,0 +1,74 @@
+const Discord = require('discord.js');
+table = require('../databaseFiles/tosReminderTable');
+const config = require('../config.json');
+
+//Adds to database
+const addToDatabase = async function(user,joinTime,reminded=false){
+    table.sync().then(async ()=>{
+        await table.create({
+            user_id: user.id,
+            joinTime: joinTime,
+            reminded: reminded
+        }).catch(err =>{
+            console.error('Tosreminder error:',err )
+        })
+    })
+}
+
+//Removes user from Database
+const removeFromDatabase = async function(user){
+    await table.destroy({
+        where:{
+            user_id:user.id
+        }
+    }).catch(err =>{
+        console.error("Tos reminder error:",err)
+    })
+}
+
+//Reminds the user
+const tosRemind = async function(client){
+    table.sync().catch(err =>{
+        console.error("Tos reminder error:",err);
+    });
+    messageEmbed = new Discord.MessageEmbed()
+     .setColor(config.colors.koaYellow)
+     .setTitle("Hey :wave: noticed you joined, but never got access to KOA.")
+     .setDescription(`Tap the check mark in <#${config.channels.tos}> to have full access to all KOA channels. Enjoy your newfound powers :relieved:`)
+     
+     const unreminded = await table.findAll({
+        where:{
+            reminded: false 
+        }
+    }).catch(err =>{
+        console.error('TosReminder error:',err)
+    });
+    unreminded.forEach(async reminder =>{
```

Comment:
unreminded can be `undefined` here - this leads to an uncaught error and the program exits.
As requested, try deleting `storageData.db` and test again, as this will replicate the state of the production server when these changes are merged.